code/game/objects/structures/xeno.dm

```diff
@@ -115,6 +115,10 @@
 	var/obj/item/clothing/mask/facehugger/hugger = null
 	var/mob/living/linked_carrier //The carrier that placed us.
 
+	var/gastrap = null
+	var/tgastier = null
```

Comment:
What does this stand for?

Reply:
ah, I added the t to the gastier to not get weird shit. The gastrap itself is for what is chosen from the list that each xeno is given.